playbooks/common/openshift-master/config.yml

```diff
@@ -199,9 +199,18 @@
       validate_checksum: yes
     with_items: masters_needing_certs
 
+- name: Inspect named certificates
+  hosts: oo_first_master
+  tasks:
+  - name: Collect certificate names
+    set_fact:
+      _named_certificates: "{{ openshift_master_named_certificates | oo_parse_certificate_names(master_cert_config_dir) }}"
```

Comment:
We should avoid variables with leading underscores to prevent issues when migrating to Ansible 2.0